### PR TITLE
B dplan 12733 fix entity manager closed

### PR DESCRIPTION
**Pull request by sierrazamudiodemos**
### Ticket
https://demoseurope.youtrack.cloud/issue/DPLAN-12733/Stellungnahme-anonymisieren-Fehler-bei-Loschen-von-Anhangen

- The anonymization of attachments means to delete all attachments of original statement
- If you try to delete all attachments from an original statement created by a private user, an error occurs because the files are linked to the user's draft statement.
- More concrete, the error is: An exception occurred while executing a query: SQLSTATE[23000]: Integrity constraint violation: 1451 Cannot delete or update a parent row: a foreign key constraint fails (`YOUR_PROJECT_DB`.`draft_statement_file`, CONSTRAINT `FK_XXXXXXXXX` FOREIGN KEY (`file_id`) REFERENCES `_files` (`_f_ident`))

### How to review/test
- In order to solve it, this PRs checks if there are draft statements related to the file. If not, then anonymization will proceed to delete the files.
- Create an stn as a private user, and then go as FP admin and select Anhänge der Stellungnahme löschen in the Anonymisierung workflow, it should work
- Create a manual stn, same workflow as before, everything should work as normal 

Delete the checkbox if it doesn't apply/isn't necessary.

- [ ] Tests updated/created
- [ ] Update documentation
- [ ] Link all relevant tickets
- [ ] Move the tickets on the board accordingly
- [ ] Data-Cy attributes added/updated ([conventions](https://dplan-documentation.demos-europe.eu/development/guidelines-conventions/coding-styleguides/twig_html.html#guideline-for-naming-cypress-hooks))

### CODE REVIEW

File: demosplan/DemosPlanCoreBundle/Logic/Statement/DraftStatementFileHandler.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ *
+ * All rights reserved
+ */
+
+namespace demosplan\DemosPlanCoreBundle\Logic\Statement;
+
+use demosplan\DemosPlanCoreBundle\Logic\FileService;
+use demosplan\DemosPlanCoreBundle\Repository\DraftStatementFileRepository;
+
+class DraftStatementFileHandler
```
Comment (mussbach) [inline demosplan/DemosPlanCoreBundle/Logic/Statement/DraftStatementFileHandler.php:16]: once in a lifetime chance to provide tests for this class from the very beginning :-)

**Review by mussbach — APPROVED** [review]
Tests would be nice, otherwise lgtm

**Review by breierDemos — APPROVED** [review]
Great, thank you!

File: demosplan/DemosPlanCoreBundle/Logic/Statement/DraftStatementFileHandler.php (hunk 1 above)
Comment (sierrazamudiodemos) [reply demosplan/DemosPlanCoreBundle/Logic/Statement/DraftStatementFileHandler.php:16]: Good idea!! I will make a PR for the tests

**Pull request merged**